docker/.env-monitor

```diff
@@ -0,0 +1,49 @@
+#
```

Comment:
The same comment as with file about. Let's modify existing file rather than create new one.